tests/cupy_tests/sparse_tests/test_csc.py

```diff
@@ -467,8 +467,28 @@ class TestCsrMatrixGetitem(unittest.TestCase):
 
     @testing.numpy_cupy_equal(sp_name='sp')
     def test_getitem_int_int(self, xp, sp):
-        return _make(xp, sp, self.dtype)[0, 1]
+        self.assertEqual(_make(xp, sp, self.dtype)[0, 1], 1)
```

Comment:
Maybe it is better to check both. In this test case, to prevent a mistake of index it is better to check its return value directly. And we also need to check if numpy result is expected.